tsconfig.json

```diff
@@ -3,9 +3,5 @@
   "compilerOptions": {
     "rootDir": ".",
     "outDir": "build"
-  },
```

Comment:
Do you remember why this change was necessary?

Reply:
Not exactly, leaving this file as is for now as it doesn't currently affect anything